client/jetpack-cloud/controller.ts

```diff
@@ -172,16 +172,13 @@ export function noSite( siteFragment: string | undefined, context: PageJSContext
 	const { getState } = context.store;
 	const currentUser = getCurrentUser( getState() ) as UserData;
 
-	if ( 0 === currentUser?.jetpack_site_count && 0 === currentUser?.atomic_site_count ) {
+	if ( 0 === currentUser?.site_count ) {
```

Comment:
Decided to add the flag `jetpack/manage-simple-sites` so that we might be able to use it throughout the Jetpack Cloud Simple development process: https://github.com/Automattic/wp-calypso/pull/89508/commits/890662f694a025b08ed035cc1a5b7569f4ce545c